docs/source/api/federations_id_deliveryservices.rst

```diff
@@ -21,7 +21,7 @@
 
 ``GET``
 =======
-Retrieves :term:`Delivery Service`\ s assigned to a federation.
+Retrieves :term:`Delivery Service`\ s assigned to a :term:`Federation`.
```

Comment:
Not necessary, but I'd appreciate it if while you're doing this if you could get rid of the `\ s`; `:term:` can handle plurals now.